noah/src/test/scala/edu/uci/ics/cloudberry/noah/kafka/TestProducer.scala

```diff
@@ -0,0 +1,74 @@
+package edu.uci.ics.cloudberry.noah.kafka
+
+import com.twitter.hbc.core.Client
+import edu.uci.ics.cloudberry.noah.GeneralProducerKafka
+import edu.uci.ics.cloudberry.noah.feed.Config
+import org.apache.kafka.clients.consumer.{ConsumerRecords, KafkaConsumer}
+import org.apache.kafka.clients.producer.{KafkaProducer, ProducerRecord}
+import org.mockito._
+import org.specs2.mock.Mockito
+import org.specs2.mutable.Specification
+import play.api.libs.ws.ahc.AhcWSClient
+import org.mockito.Mockito._
+import com.twitter.hbc.core.endpoint.Location
+
+class TestProducer extends Specification with Mockito {
+
+  "FileProducer" should {
+    "--  read from a .gz file and send records to kafka" in {
```

Comment:
Furthermore, if you take a look at the current output example from [Travis](https://travis-ci.org/ISG-ICS/cloudberry/builds/157386982), i don't see the current format is not good.